src/Microsoft.PowerPlatform.PowerApps.Persistence/Yaml/ControlPropertiesCollectionConverter.cs

```diff
@@ -41,12 +41,15 @@ public void WriteYaml(IEmitter emitter, object? value, Type type)
 
         emitter.Emit(new MappingStart());
 
-        foreach (var kv in collection)
+        var sortedKeys = collection.Keys.OrderBy(x => x);
```

Comment:
Should we specify culture for sorting?

Reply:
good catch. `StringComparer.CurrentCultureIgnoreCase` ?

Reply:
Yeah, probably an Ordinal comparison.
I'm not sure what DocumentServer is doing for culture settings, but there could very well be a per-thread culture in response to requests.